saleor/graphql/account/mutations/staff/customer_update.py

```diff
@@ -162,3 +166,24 @@ def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
 
         # Return the response
         return cls.success_response(new_instance)
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance, cleaned_input):
+        manager = get_plugin_manager_promise(info.context).get()
+
+        default_addresses = cls.save_and_apply_addresses_from_input(
+            cleaned_input=cleaned_input, user_instance=instance
+        )
+
+        super().save(info, instance, cleaned_input)
```

Comment:
Same here with saving the whole object instead of changed fields. 
Also the save is called twice, in this line and in 187 and can probably be called once